mut/redirects/redirect_main.py

```diff
@@ -89,219 +89,201 @@ def write_to_file(rules: List[RuleDefinition], output_path: str) -> None:
             f.write('\n')
 
 
+def parse_line(line, rc, line_num, version_regex, url_regex, dict_regex):
```

Comment:
Type annotations

```
from typing import Pattern
...
def parse_line(line: str, rc: RedirectContext, line_num: int,
               version_regex: Pattern, url_regex: Pattern, dict_regex: Pattern) -> None:
```

Reply:
Done. Also removing the dict_regex value / param because it doesn't actually seem to be used anywhere.